README.md

```diff
@@ -208,7 +210,15 @@ programs.vscode = {
 };
 ```
 
+### Vim
+
+- Experimental [CoqTail](https://github.com/whonore/Coqtail) support by Wolf Honore: 
+  https://github.com/whonore/Coqtail/pull/323
+
+  See it in action https://asciinema.org/a/mvzqHOHfmWB2rvwEIKFjuaRIu
+
 ### Neovim
```

Comment:
I would suggest merging the vim entries since there is little difference between vim and neovim (aside from configuration).

Reply:
I understand they are different projects, right?

Reply:
Moreover I also understand that LSP support is very different between them, right?